mir-module/src/main/resources/META-INF/resources/js/mir/geo-coords.js

```diff
@@ -1,132 +1,202 @@
 (function($) {
-  $﻿(document).ready(function() {
-    $('.show_openstreetmap').click(function() {
-		if (!$(this).hasClass("map_drawn")) {
-			drawMap($(this));
-		}
-		$(this).parent().next(".openstreetmap-container").collapse('toggle');
-    });
+    $﻿(document).ready(function() {
+        $('.show_openstreetmap').click(function() {
+            $(this).parent().next(".openstreetmap-container").collapse('toggle');
+            if(!$(this).hasClass("map_drawn")) {
+                drawMap($(this));
+            }
+        });
+
+        function drawMap(btn) {
+            var input = $(btn).parent().parent().prev().find("input[name*='coordinates']")[0];
+            var typeSelect = $(btn).parent().next().find("#type")[0];
+            var mapElement = $(btn).parent().next().find(".map")[0];
+            // latitude, longitude
+            var lat = 50.930453, lon = 11.587786;
```

Comment:
this should be configurable properties. Is it possible to use mycore.properties here?